os/pm/pm_timedstay.c

```diff
@@ -108,9 +108,13 @@ static void timer_timeout(int argc, uint32_t state, uint32_t pid)
  *
  ************************************************************************/
 
-int pm_timedstay(enum pm_state_e state, unsigned int timer_interval)
+int pm_timedstay(enum pm_state_e state, unsigned int milliseconds)
 {
 	int pid = PIDHASH(getpid());
+	int timer_interval;
+
+	/* Converting from milliseconds to ticks */
+	timer_interval = ((milliseonds * CLOCKS_PER_SEC) /  1000);
```

Comment:
1. Should include necessary header for `CLOCKS_PER_SEC`
2. Regardless of this change, this file uses some config, but no including <tinyara/config.h>
3. How about using `MSEC2TICK`?

Reply:
2. it was already present in pm.h header . Anyway , i have added it now
3. updated

Reply:
2. It should not have any dependency with other header file. If we use no config in the pm.h, it's much difficult to know whether we can remove including config.h there or not. Let's include necessary header at the file.